universum/modules/output/output.py

```diff
@@ -70,3 +73,21 @@ class HasOutput(Module):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.out: Output = self.out_factory()
+
+
+class MinimalOut:
+    def __init__(self, silent=False):
+        self.silent = silent
```

Comment:
Please note that default behaviour of old MinimalOut used to be not to print anything (silent = True).
I think it's fine to have silent=False as default, just making sure the change of default behaviour is intentional.

Reply:
I agree that silent=False by default is more natural than the previous behavior.

Reply:
Yes, I've changed it because I also find it more natural.